### PR TITLE
Add method to update GeoQuery center and radius together atomically. Fixes #39

### LINKED ISSUE #39: Add method to update GeoQuery center and radius together atomically
I have been calling GeoQuery setCenter() and setRadius() as user zooms in/out in google maps (in android app) and noticed that that seems to be kicking off parallel sets of firebase queries (they both call setupQueries()).  Should there be single method to set both of these (similar to geoFire.queryAtLocation(() method when initially creating the GeoQuery object)?

### PR DESCRIPTION
**Pull request by joreilly**
Fixes https://github.com/firebase/geofire-java/issues/39.

### CODE REVIEW

File: src/main/java/com/firebase/geofire/GeoQuery.java [hunk 1]
```diff
@@ -411,4 +411,17 @@ public synchronized void setRadius(double radius) {
             this.setupQueries();
         }
     }
+
+    /**
+     * Sets the center and radius (in kilometers) of this query, and triggers new events if necessary.
+     * @param radius The new radius value of this query in kilometers
```
Comment (jwngr) [inline src/main/java/com/firebase/geofire/GeoQuery.java]: Can you please also add an `@param` line for `center` before this line?

`* @param center The new center`

**Comment by jwngr** [comment]
This looks good to me outside of one comment, but I'm going to assign to @jdimond and @samtstern for one of them to give the final approval since I'm not a big Java developer.

**Comment by joreilly** [comment]
Param comment added

**Comment by samtstern** [comment]
This LGTM.  @jwngr would this be version 2.1.0 then?

**Comment by jwngr** [comment]
Yup, `2.1.0` is correct. We just need to merge this in, update the root `changelog.txt`, add this to the docs, and publish the new version.

**Comment by samtstern** [comment]
Ok I am going to merge this, I will follow up with a CL that changes the changelog and anything else release-y.

@joreilly thanks for the contribution!

**Comment by samtstern** [comment]
@joreilly @jwngr Version `2.1.0

**Comment by chiragpatel3131** [comment]
i want to remove old markers of users location on google map but the 
Geoquery.setCenter or Geoquery.setLocation both don't work when user A is moving

if user A Is the app user tracking nearby users 
if B user is moving out of radius or in to radius A gets update on google map when A is static

but if B is not moving and A IS Moving than the B don't get automatically exited 
i means at that time onKeyExited don't notify that the A has new center location.

any solution for that ?

**Pull request merged**